src/css/responsive/tablet.styl

```diff
@@ -60,6 +60,33 @@ svg.pagination-arrow
 
 // ******************************************************
 
+// BLOG
+
+.blog-item-single
+  max-width: none
+
+body#about
```

Comment:
then it shouldnt be the ID of `about` IDs should be unique not just on the page but across a project